src/pgduckdb.cpp

```diff
@@ -131,6 +133,14 @@ DuckdbInitGUC(void) {
 	                     "Allow DuckDB to load extensions with invalid or missing signatures",
 	                     &duckdb_allow_unsigned_extensions, PGC_SUSET);
 
+	DefineCustomVariable("duckdb.autoinstall_known_extensions",
+	                     "Whether known extensions are allowed to be automatically installed when a DuckDB query depends on them",
+	                     &duckdb_autoinstall_known_extensions, PGC_POSTMASTER, GUC_SUPERUSER_ONLY);
+
+	DefineCustomVariable("duckdb.autoload_known_extensions",
+	                     "Whether known extensions are allowed to be automatically loaded when a DuckDB query depends on them",
+	                     &duckdb_autoload_known_extensions, PGC_POSTMASTER, GUC_SUPERUSER_ONLY);
```

Comment:
I think PGC_SUSET is fine for these. It seems alright to change these after startup (as long as you're superuser).
```suggestion
	                     &duckdb_autoinstall_known_extensions, PGC_SUSET);

	DefineCustomVariable("duckdb.autoload_known_extensions",
	                     "Whether known extensions are allowed to be automatically loaded when a DuckDB query depends on them",
	                     &duckdb_autoload_known_extensions, PGC_SUSET);
```